app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -356,15 +356,16 @@ public function compareDateValue($lead, $field, $value)
         $q->select('l.id')
             ->from(MAUTIC_TABLE_PREFIX.'leads', 'l')
             ->where(
-                $q->expr()->andX(
+                $q->expr()->and(
```

Comment:
@escopecz I made this on purpose, I read your comments regarding the replacement of `andX` with `and` on Jira but I do not think this has a huge impact

Reply:
Thanks for pointing this out. It's OK to replace it in methods where we are certain that only the DBAL query builder is used.